### DISCUSSION TITLE
How to remap keys in the 'space' mode?

### DISCUSSION THREAD

**Discussion by amstaniec** [helix-editor/helix#4633] [Q&A] [answered]
### Discussed in https://github.com/helix-editor/helix/discussions/2844

<div type='discussions-op-text'>

<sup>Originally posted by **ipstone** June 20, 2022</sup>
Hello, 
Coming from vim, I usually remap` <space>`+z to :quit - 
is there a way to do so in helix?

I tried to add the following in config.toml , but helix would not accept it:

[keys.space]
z = 'quit'
</div>

I tried liek this:
```
[keys.normal.space]
"space" = ":command_mode"
```

but that gives me:

```
❯ hx
Bad config: data did not match any variant of untagged enum KeyTrie for key `keys.normal` at line 25 column 1
Press <ENTER> to continue with default config
```

**Comment by the-mikedavis** [comment, marked as answer]
The `:` is used for typable commands - ones that you can type out in the prompt in helix with `:<command-name>` and use kebab-case. `command_mode` is a regular command so you can bind to the command directly like so:

```toml
[keys.normal.space]
space = "command_mode"
```
